src/background/panel/openEnablePanel.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 Centrality Investments Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import queryString from 'query-string';
+import { browser } from 'webextension-polyfill-ts';
+import {EnableCommand} from '../../types/message';
+
+export default (params: { noheader: boolean; enable: EnableCommand }) => {
```

Comment:
looks like we can use one openPanel() for all of them

Reply:
Thanks for the review Ian. Will change this:D